pyproject.toml

```diff
@@ -14,11 +14,14 @@ pytest = "^5.2"
 black = "^21.11b0"
 poethepoet = "^0.11.0"
 coverage = "^6.1.2"
+starknet-devnet = "^0.1.9"
+aiounittest = "^1.4.1"
 
 [tool.poe.tasks]
 test = "coverage run -m pytest src"
 test_report = "coverage report"
 test_html = { shell = "coverage html && open ./htmlcov/index.html" }
+test_e2e = { shell = "starknet-devnet --host localhost --port 5000 &; DEVNET_PID=$! && sleep 1 && pytest src/e2e; kill $DEVNET_PID" }
```

Comment:
We should run e2e tests with coverage as well.
I suggest we make `test` to run both unit tests and e2e tests and separately `test_unit` for unit tests and `test_e2e` for only e2e tests.

Reply:
I just want to be able to generate a coverage report from all our tests.